tests/tests/datm_ocn_ice_mx025_cfsr

```diff
@@ -0,0 +1,63 @@
+#
+#  CPLD_DATM_MOM6_CICE_CMEPS_1_STEP_COLD test
```

Comment:
Please add CFSR to indicate this test uses CFSR forcings.

Reply:
Done.